test/fixtures/everything_at_once/results.json

```diff
@@ -17,7 +17,7 @@
         {
             "name": "third test",
             "status": "error",
-            "message": "\nStacktrace:\n [1] error(::String) at ./error.jl:33\n [2] top-level scope at ./runtests.jl:14\n [3] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Test/src/Test.jl:1113\n [4] top-level scope at ./runtests.jl:14\n",
+            "message": "\nStacktrace:\n [1] error(::String) at ./error.jl:33\n [2] top-level scope at ./runtests.jl:14\n [3] top-level scope at [...]/julia/stdlib/v1.4/Test/src/Test.jl:1113\n [4] top-level scope at ./runtests.jl:14\n",
```

Comment:
The last bit should stay so that the student could find that file if necessary. It's not important in this particular example, but if they use built-in functions, they may need to look at them to find out why they're erroring

Reply:
I agree